src/features/auth/context/AuthContext.tsx

```diff
@@ -38,11 +39,18 @@ export const useConnectServicesRequiringUserId = (): ((accessToken: string | nul
     (accessToken) => {
       if (!accessToken) return
 
-      const userId = getUserIdFromAccesstoken(accessToken)
+      const userId = getUserIdFromAccessToken(accessToken)
       if (userId) {
         BatchUser.editor().setIdentifier(userId.toString()).save()
         firebaseAnalytics.setUserId(userId)
         eventMonitoring.setUser({ id: userId.toString() })
+
+        const accessTokenExpiration = decodeToken(accessToken)?.exp
+        eventMonitoring.setExtras({
+          accessTokenExpirationDate: accessTokenExpiration
+            ? getDateFromTimestampInSeconds(accessTokenExpiration).toISOString()
+            : "can't get access token expiration date",
```

Comment:
quand j'ai dit qu'une fonction pourrait etre chouette

j'imaginais qu'elle prendrait directement le JWT en entrée 

```suggestion
        eventMonitoring.setExtras({
          accessTokenExpirationDate: toto(accessToken),
```

ou peut etre

```suggestion
        eventMonitoring.setExtras({
          accessTokenExpirationDate: toto(accessToken),
```

si on veut pas avoir une string d'erreur générique (de toute façon, ça sera dans une clé spécifique, donc je pense qu'osef)

Reply:
```ts
type DateISO = string
const tutu = "can't get token expiration date"
const toto = (jwt: JWT) => DateISO | typeof tutu
```